extract_unique_sensors_01.py

```diff
@@ -0,0 +1,77 @@
+from urllib.request import urlopen
+from pathlib import Path
+
+file_name = "SensorList.txt"
+url = "http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt"                                                                          
+kat_sensor = "kat.sensor"
+
+def read_sensors(file_name, url):
+    """This function opens, reads and extracts the contents of the file_name.
```

Comment:
Please update the docstring, as you are trying to retrieve from URL with failover of reading the filename.